pkg/geoipupdate/vars/version.go

```diff
@@ -2,4 +2,4 @@
 package vars
 
 // Version defines current geoipupdate version.
-const Version = "4.11.1"
+const Version = "5.0.0"
```

Comment:
There is no harm in doing this, but the release script does update this version number to match the change log when a release is done.